### PR TITLE
Fix: Post Masonary - Fixed the JS UAGBPostMasonry is not defined error.

### PR DESCRIPTION
**Pull request by AbhijitNage123**
### Description

Trello - https://trello.com/c/RrMa2Lit/278-fix-post-masonry-fixed-the-js-uagbpostmasonry-is-not-defined-error

### Screenshots

Step to reproduce -
- Activate only Masonry block.
- Drag and drop post masonry update the page and check the console on the front-end.

### Types of changes

- Bug fix (non-breaking change which fixes an issue)

### How has this been tested?

- Activate only Masonry block.
- Drag and drop post masonry update the page and check the console on the front-end.
- Activate all blocks and check the console again.

### Checklist:
- [x] My code is tested
- [ ] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [x] I've included any necessary tests 
- [ ] I've included developer documentation 
- [x] I've added proper labels to this pull request

### CODE REVIEW

File: classes/class-uagb-config.php [hunk 1]
```diff
@@ -1589,7 +1589,7 @@ public static function get_block_attributes() {
 						'description' => __( 'This block fetches the blog posts you may have on your website and displays them in a masonry layout.', 'ultimate-addons-for-gutenberg' ),
 						'default'     => true,
 						'extension'   => false,
-						'js_assets'   => array( 'uagb-masonry', 'uagb-imagesloaded', 'uagb-post-js' ),
+						'js_assets'   => array( 'uagb-masonry', 'uagb-imagesloaded', 'uagb-post-js', 'uagb-slick-js' ),
```
Comment (RajkiranBagal) [inline classes/class-uagb-config.php:1592]: @AbhijitNage123 We do not need to add slick-js dependency for Masonry. Just add the dependency for uagb-post-js of slick-js conditionally only for Carousel. As we discussed on call.

Comment (AbhijitNage123) [reply classes/class-uagb-config.php]: Done

File: classes/class-uagb-config.php [hunk 2]
```diff
@@ -2940,7 +2943,7 @@ public static function get_block_assets() {
 					),
 					'uagb-post-js'           => array(
 						'src' => UAGB_URL . 'assets/js/post.js',
-						'dep' => array( 'jquery', 'uagb-slick-js' ),
+						'dep' => $slickJs,
```
Comment (RajkiranBagal) [inline classes/class-uagb-config.php:2946]: Do not use camelCase variable in PHP.

File: ~ [hunk 3]
```diff
@@ -0,0 +1,6 @@
+Merge branch 'next-release' of https://github.com/brainstormforce/ultimate-addons-for-gutenberg into post-masonary-not-defined-issue
+# Please enter a commit message to explain why this merge is necessary,
```
Comment (RajkiranBagal) [inline ~:2]: Is this change needed in this PR?

Comment (AbhijitNage123) [reply ~:2]: I just pull the next release and push the code.

File: classes/class-uagb-config.php (hunk 2 above)
Comment (AbhijitNage123) [reply classes/class-uagb-config.php]: Done

File: classes/class-uagb-config.php [hunk 4]
```diff
@@ -2912,6 +2912,9 @@ public static function get_block_attributes() {
 		 */
 		public static function get_block_assets() {
 
+			$blocks  = UAGB_Admin_Helper::get_block_options();
+			$slickjs = ( ( false === $blocks['uagb/post-carousel']['is_activate'] ) ? array( 'jquery' ) : array( 'jquery', 'uagb-slick-js' ) );
```
Comment (RajkiranBagal) [inline classes/class-uagb-config.php:2916]: Can we use a descriptive & relevant variable name?

File: ~ (hunk 3 above)
Comment (RajkiranBagal) [reply ~:2]: This is an unnecessary change, please remove it.

**Review by RajkiranBagal — CHANGES_REQUESTED** [review]

File: classes/class-uagb-config.php (hunk 2 above)
Comment (RajkiranBagal) [reply classes/class-uagb-config.php]: PHP variable names should be like this `$slick_js`

File: classes/class-uagb-config.php (hunk 4 above)
Comment (AbhijitNage123) [reply classes/class-uagb-config.php]: Done

File: ~ (hunk 3 above)
Comment (AbhijitNage123) [reply ~]: Removed

File: classes/class-uagb-config.php (hunk 2 above)
Comment (AbhijitNage123) [reply classes/class-uagb-config.php]: Updated

**Review by RajkiranBagal — APPROVED** [review]

**Review by sandeshjangam — APPROVED** [review]
